### ISSUE TITLE
Not support Vast 3.0

### ISSUE THREAD

**Issue by JuliaTeasernet** [fluid-player/fluid-player#124]
In our old version of Vast all requirements of Vast 2.0 are met
Example http://site.teasernet.com/lila/vast2.xml 
The player is playing correctly
Example http://site.teasernet.com/fluidnew.html

We've added support CDATA and tag <AdSystem>
Example http://site.teasernet.com/lila/vast3.xml
The player does not launch a new version of the vast
Example http://site.teasernet.com/fluidnew2.html

[Error] (401): Failed to load Vast

Vast is not correctly parsed. Please check

**Comment by cian-exads** [comment]
Hi @JuliaTeasernet, this seems to be occurring due to the whitespace surrounding the CDATA, as Fluid Player does indeed support the CDATA itself. I'm going to add in a fix for this so that the whitespace won't cause the error to occur. I'll update this ticket with the status of this fix.

**Comment by golikov** [comment]
This was fixed as a part of v2.2.0

**Comment by JuliaTeasernet** [comment]
After fix Vast can't playing: http://site.teasernet.com/fluidnew2.html
version 2.2.0

**Comment by kristijanexads** [comment]
Hi @JuliaTeasernet , it seems that there is a similar problem as before, trimming whitespaces between CDATA solved previous issue, but we will have to apply the same on XML nodes around CDATA, so until we release this fix we suggest to change nodes around CDATA without whitespaces:  

from:

```
<MediaFile delivery="progressive" type="video/mp4" width="1280" height="720" scalable="true" maintainAspectRatio="true">
    <![CDATA[
        http://api2.jiouv.com/v/vbc/bc091cf319b61e8cd4dc/video.mp4
    ]]>
</MediaFile>
```

to:
```
<MediaFile delivery="progressive" type="video/mp4" width="1280" height="720" scalable="true" maintainAspectRatio="true"><![CDATA[http://api2.jiouv.com/v/vbc/bc091cf319b61e8cd4dc/video.mp4]]></MediaFile>
```

**Comment by JuliaTeasernet** [comment]
How long will fix take?

**Comment by cian-exads** [comment]
@JuliaTeasernet we hope to release a fix for this soon. @kristijanexads has recommended a temporary fix for now, hopefully this will resolve the issue for now.

**Comment by JuliaTeasernet** [comment]
@cian-exads we try

vast http://site.teasernet.com/vast3.xml (without whitespaces )
player http://site.teasernet.com/fluidnew3.html

Does not work

**Comment by cian-exads** [comment]
The fix is now live as of v2.2.1 @JuliaTeasernet
